docs/src/intro-python.md

```diff
@@ -40,12 +40,13 @@ pip install pytest-playwright
 </TabItem>
 <TabItem value="anaconda">
 
-Install the [Pytest plugin](https://anaconda.org/Microsoft/pytest-playwright):
+Install Playwright and the [Pytest plugin](https://anaconda.org/Microsoft/pytest-playwright):
 
 ```bash
 conda config --add channels conda-forge
 conda config --add channels microsoft
 conda install playwright
```

Comment:
We can remove this now, since the pytest plugin depends on playwright

Reply:
Thanks, done.